homeassistant/components/life360/button.py

```diff
@@ -19,12 +19,12 @@ async def async_setup_entry(
     coordinator: Life360DataUpdateCoordinator = hass.data[DOMAIN].coordinators[
         config_entry.entry_id
     ]
-    for member_id, member in coordinator.data.members.items():
-        async_add_entities(
-            [
-                Life360UpdateLocationButton(coordinator, member.circle_id, member_id),
-            ]
-        )
+    async_add_entities(
+        [
```

Comment:
I think we can drop the list and just pass a generator expression without extra brackets.